### PR TITLE
azure: Fix SRIOV test VERIFY-IFUP-NICS-SRIOV

### PR DESCRIPTION
**Pull request by ader1990**
Fixes issue: https://github.com/LIS/LISAv2/issues/137

### CODE REVIEW

File: Libraries/Test-Framework.psm1 [hunk 1]
```diff
@@ -200,23 +200,36 @@ function Collect-TestLogs {
...
         RemoteCopy -download -downloadFrom $PublicIP -downloadTo $LogsDestination `
              -Port $SSHPort -Username "root" -password $Password `
-             -files "/home/${Username}/state.txt, /home/${Username}/${TestName}_summary.log"
-        $statePath = Join-Path $LogDir "state.txt"
-        $testResult += $(Get-Content $statePath)
-        $testResult = $resultTranslation[$testResult]
+             -files $filesTocopy
+        $summary = Get-Content (Join-Path $LogDir "Summary.log")
+        $testState = Get-Content (Join-Path $LogDir "state.txt")
+        $testResult = $resultTranslation[$testState]
```
Comment (juhlee-microsoft) [inline Libraries/Test-Framework.psm1]: Line number 224 shows 'append' but here just 'replacement'. Are they different way to collect to $testResult?

**Comment by ader1990** [comment]
@juhlee-microsoft I have addressed your comments.

**Comment by juhlee-microsoft** [comment]
@ader1990 I can not see your reply or answer of my question here. what did you address in my comment?

File: Libraries/Test-Framework.psm1 (hunk 1 above)
Comment (ader1990) [reply Libraries/Test-Framework.psm1]: I have addressed your comment

**Comment by chvalean** [comment]
@juhlee-microsoft can you please also sign-off on this PR and merge it?
This will resume the functionality of the latest images pipeline.

Adrian modified the code based on your comment, so for that reason it's showing up as outdated.

**Comment by juhlee-microsoft** [comment]
Thanks, Chris, for explanation. Signed off


________________________________

**Pull request merged**